include/reliable_updates.h

```diff
@@ -0,0 +1,215 @@
+#pragma once
+
+#include <color_spinor_field.h>
+#include <blas_quda.h>
+#include <util_quda.h>
+
+struct reliable_updates {
```

Comment:
Style convention: I think this should be `ReliableUpdates`

Reply:
Otherwise---I really like how reliable updates have been abstracted into a separate `struct`.

Reply:
https://github.com/lattice/quda/pull/1061/commits/aca1ee4c94659aab2be39a304cdd6c32c6408ab9